helix-view/src/editor.rs

```diff
@@ -342,6 +346,9 @@ pub struct Config {
         deserialize_with = "deserialize_alphabet"
     )]
     pub jump_label_alphabet: Vec<char>,
+    /// Whether to prepend a comment token onto a new line that follows a commented line. (default: false)
+    #[serde(default = "default_continue_comments")]
```

Comment:
```suggestion
```

This doesn't need a default directive for serde since `Default for Config` is implemented manually.

Also the doc here is outdated that the default is false. This should also be added to the book

I wonder if this needs to be configurable at all? I'm not sure it's configurable in Kakoune for example

Reply:
> I wonder if this needs to be configurable at all? I'm not sure it's configurable in Kakoune for example

hm... I kinda agree here. I thought that there could be some people who don't like to have this feature but I can't get up an idea why anyone would dislike this. I can remove this config option then. It should be simple to add it back.